### DISCUSSION TITLE
serving an image generated by thinstation

### DISCUSSION THREAD

**Discussion by paykocaydin** [ltsp/ltsp#664] [Q&A]
Hi, 

We do have a working LTSP setup. (thans to Alkis and Shrenik) 
The image we serve is generated by "ltsp image /" (chrootless) 

What we need is a very small image just to connect to a windows server using rdp. We do not want users to login to Linux, what we are trying to achieve is that the users will see Windows login screen when the thin client boots. 

We have build an image for this purpose using thinstation enviroment. It has created all the necessary files for PXE/PXELINUX boot. 

We were not able to find any document on how to integrate these files with LTSP. 
We even do not know if this can be done.

Can someone point me to a document where we can find this information. 

Regards,

Aydin

**Comment by intelliant01** [comment]
This should be helpful - https://ltsp.org/man/ltsp-kernel/
